plugin-packs/postcss-preset-env/src/lib/get-unsupported-browsers-by-feature.js

```diff
@@ -1,26 +1,41 @@
-import * as caniuse from 'caniuse-lite';
+const browsers = [
+	'ie',
+	'edge',
+	'firefox',
+	'chrome',
+	'safari',
+	'opera',
+	'ios_saf',
+	'android',
+	'op_mob',
+	'and_chr',
+	'and_ff',
+	'and_uc',
+	'samsung',
+	'and_qq',
+	'baidu',
+	'kaios',
+];
 
 // return a list of browsers that do not support the feature
 export default function getUnsupportedBrowsersByFeature(feature) {
-	const caniuseFeature = caniuse.features[feature];
-
 	// if feature support can be determined
-	if (caniuseFeature) {
-		const stats = caniuse.feature(caniuseFeature).stats;
+	if (feature.browser_support) {
+		const query = [];
+
+		browsers.forEach(browser => {
+			const browserSupport = feature.browser_support[browser];
+			// If the version is something like TP we can't do < TP
+			const isValid = typeof browserSupport === 'string' && browserSupport.match(/^[0-9]+$/) !== null;
 
-		// return an array of browsers and versions that do not support the feature
-		const results = Object.keys(stats).reduce(
-			(browsers, browser) => browsers.concat(
-				Object.keys(stats[browser]).filter(
-					version => stats[browser][version].indexOf('y') !== 0,
-				).map(
-					version => `${browser} ${version}`,
-				),
-			),
-			[],
-		);
+			if (isValid) {
+				query.push(`${browser} < ${feature.browser_support[browser]}`);
+			} else {
+				query.push(`${browser} >= 1`);
```

Comment:
What is the current behaviour if you pass browser to postcss-preset-env that doesn't have stats for a certain feature?

Is it enabled or disabled?

Reply:
This happens with `custom-properties`, `ie` does not appear so that means every IE version is added to the "unsupported list". If the list matches any `ie` it will be polyfilled.

Browser support for custom-properties:

```json
"browser_support": {
  "edge": "16",
  "firefox": "31",
  "chrome": "49",
  "safari": "10",
  "opera": "36",
  "ios_saf": "10",
  "android": "97",
  "op_mob": "64",
  "and_chr": "97",
  "and_ff": "95",
  "and_uc": "12.12",
  "samsung": "5",
  "kaios": "2.5"
}
```

Test:

```
'custom-properties': {
	message: 'supports { browsers: "ie >= 10" } usage',
	options: {
		browsers: 'ie >= 10'
	}
}
```

Reply:
Thank you for checking this!